WebDriverAgentTests/IntegrationTests/FBScrollingTests.m

```diff
@@ -80,4 +82,20 @@ - (void)testFarScrollToVisible
   FBAssertVisibleCell(cellName);
 }
 
+- (void)testAttributeWithNullScrollToVisible
+{
+  NSError *error;
+  NSArray<XCUIElement *> *queryMatches = [self.testedApplication fb_descendantsMatchingClassChain:@"XCUIElementTypeWindow[1]/XCUIElementTypeOther[1]/XCUIElementTypeOther[1]/XCUIElementTypeOther[1]/XCUIElementTypeTable[1]/XCUIElementTypeCell[60]" shouldReturnAfterFirstMatch:NO];
```

Comment:
this class chain can be replaced with `**/XCUIElementTypeTable/XCUIElementTypeCell[60]`

Reply:
OK